app.js

```diff
@@ -1,21 +1,29 @@
-(function () {
+(function a() {
 
-    $(document).on("DOMContentLoaded", init, event);
+    $(init);
     var table;
     var cellIdArray = [];
+    var randomArray;
     var counter;
+    var openCellsCounter = 0;
 
     function init() {
         table = $('#table');
         showTable();
-
-        var button = $('#button');
-        button.on('click', switchTableColors);
-        $('table').on('dblclick', 'td', changeColor);
         $('table').on('click', 'td', onCellClick);
-
     }
 
+    $(document).ready(function () {
+        document.oncontextmenu = function () {

```

Comment:
I don't think we need to disable contextmenu for whole document. Calling event.preventDefault in td.contextclick handler should be enough.